docs/cloud-batch.md

```diff
@@ -1,31 +1,32 @@
-# Cloud Batch in the HPC Toolkit
+# Google Cloud Batch in the HPC Toolkit
 
-Using Cloud Batch with the HPC Toolkit simplifies the setup needed to provision
-and run more complex scenarios, for example, setting up a shared file system and
-installing software to be used by Batch jobs. It also makes it possible to share
-tested infrastructure solutions that work with Cloud Batch via HPC Toolkit
-blueprints.
+Using Google Cloud Batch with the HPC Toolkit simplifies the setup needed to
+provision and run more complex scenarios, for example, setting up a shared file
+system and installing software to be used by Google Cloud Batch jobs. It also
+makes it possible to share tested infrastructure solutions that work with Google
+Cloud Batch via HPC Toolkit blueprints.
 
 ## Modules
 
 The HPC Toolkit supports Google Cloud Batch through two Toolkit modules:
 
 - [cloud-batch-job](../community/modules/scheduler/cloud-batch-job/README.md):
-  - Generates a Cloud Batch job template that can be submitted to the Cloud
-    Batch API
-  - Creates an instance template for the Cloud Batch job to use
-  - Works with existing Toolkit modules such as `vpc`, `filestore`, `startup-script` & `spack`
+  - Generates a Google Cloud Batch job template that can be submitted to the
+    Google Cloud Batch API
+  - Creates an instance template for the Google Cloud Batch job to use
+  - Works with existing Toolkit modules such as `vpc`, `filestore`,
+    `startup-script` & `spack`
```

Comment:
Missed this on my last review: `spack-install`, not `spack`

Reply:
Done.